src/ScheduleFunctions.cpp

```diff
@@ -1021,6 +1021,8 @@ bool validate_schedule(Function f, Stmt s, const Target &target, bool is_output,
             << "Func " << f.name() << " is scheduled inline, so it"
```

Comment:
It looks like this check for specializations should also go into the validation function